### PR TITLE
Migration to Make Users Confirmable with Devise

### PR DESCRIPTION
**Pull request by BGRicker**
This migration will make prepare the users model for confirmable users with Devise: https://github.com/EBWiki/EBWiki/issues/2631

The migration now reflects the exact one listed in the issue rather than editing it to reflect the guides 👍

In your PR did you:

  - [√] Include a description of the changes?
  - [√] Mention the issue the PR addresses?
  - [ ] Include screenshots of any changes to the UI?
  - [ ] Isolate any changes to gems (meaning that any new, updated, or removed gems and resulting code changes should be in their own PR)?
  - [ ] Add and/or update specs for your code?

### CODE REVIEW

**Comment by rlgreen91** [comment]
Please merge master into this branch.

File: db/structure.sql [hunk 1]
```diff
@@ -1665,3 +1675,6 @@ INSERT INTO schema_migrations (version) VALUES ('20181013103240');
 
 INSERT INTO schema_migrations (version) VALUES ('20181018123139');
 
+INSERT INTO schema_migrations (version) VALUES ('20181025220728');
```
Comment (d4mk0) [inline db/structure.sql]: Hi @BGRicker 

Maybe my question will seem silly, but why this line was added here?
Migration with timestamp `20181025220728` doesn't existed in this tree https://github.com/EBWiki/EBWiki/tree/29621b6de008309a7b4acc57d4ad865c680d9714/db/migrate

Comment (BGRicker) [reply db/structure.sql]: hey @d4mk0, not a silly question at all, this was a good catch. It was left over from a previous version of my migration, it should have been removed so thank you for noticing it!

**Comment by rlgreen91** [comment]
Please merge master into this branch.

**Pull request merged**